pkg/cmd/release/git.go

```diff
@@ -286,8 +291,16 @@ func findHealthyBuild(potentialRefs []string) (buildInfo, error) {
 	return buildInfo{}, fmt.Errorf("no ref found")
 }
 
-// listRemoteBranches retrieves a list of remote branches using a pattern, assuming the remote name is `origin`.
-func listRemoteBranches(pattern string) ([]string, error) {
+// IsStagingBranch returns true if branch follows one of the following patterns:
```

Comment:
I wasn't sure of the best "home" for this method, so feel free to suggest a better file/package for this?

i.e. this isn't a "git" method, but it has knowledge about release/staging branches